erpnext/templates/pages/cart.html

```diff
@@ -79,6 +79,9 @@ <h2>{{ _("My Cart") }}</h2>
 		<div id="cart-fields">
 			{% for f in quotation_fields %}
 			<div class="cart-field-data hidden" data-fieldname="{{ f }}">{{ doc.get(f) or '' }}</div>
+			{% if f == 'delivery_date' %}
+			<div class="cart-field-data hidden" data-fieldname="day">{{ doc.get(f).strftime('%A') or '' }}</div>
+			{% endif %}
```

Comment:
If you're gonna send the Day of the Week from server... Then include it as another field that is sent on every update and not just on page reload